### DISCUSSION TITLE
"language" attribute of the "TextArea" widget is not working

### DISCUSSION THREAD

**Discussion by Kalebe16** [Textualize/textual#4358] [Q&A]
## This is my code(I copied from the official documentation):
```python
from textual.app import App, ComposeResult
from textual.widgets import TextArea

TEXT = """\
def hello(name):
    print("hello" + name)

def goodbye(name):
    print("goodbye" + name)
"""


class TextAreaExample(App):
    def compose(self) -> ComposeResult:
        yield TextArea.code_editor(TEXT, language="python")



if __name__ == "__main__":
    app = TextAreaExample()
    app.run()

```
## Command used
![image](https://github.com/Textualize/textual/assets/87911732/8acbf145-c4be-4956-95f1-8135e238414d)

## This is the result
![image](https://github.com/Textualize/textual/assets/87911732/0d512de0-4b63-40f7-83e9-5c31a183a0fa)

## Extra observations
- My operating system: **Ubuntu**
- My terminal emulator: **Tabby**(I also tested it on the standard Ubuntu terminal)
- My python package manager: **pip**
- My python version: **3.12.1**
- I am using an isolated virtual environment

**Comment by TomJGooding** [comment]
This is explained at the top of the TextArea docs: https://textual.textualize.io/widgets/text_area/#syntax-highlighting-dependencies
